content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -8,26 +8,33 @@ menu:
         weight: 1
 ---
 
-This page provides an overview of how to use the Pulumi plugin for Spinnaker to run Pulumi apps. Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
-CI/CD system. So the steps described here can be altered to fit into any existing type of deployment setup.
+This page provides an overview of how to use the Pulumi plugin for Spinnaker to run Pulumi apps. Pulumi doesn't require
+any particular arrangement of stacks or workflow to work in a CI/CD system. So the steps described here can be altered
+to fit into any existing type of deployment setup.
 
 ## Prerequisites
 
 ### Spinnaker installation
 
 > [Spinnaker Plugin Creator's Guide](https://www.spinnaker.io/guides/developer/plugin-creators/overview/).
 
-Plugins are a recent addition to Spinnaker and as such require a more recent version to use plugins as well as the Pulumi plugin for Spinnaker, which this guide will show you how to use.
+Plugins are a recent addition to Spinnaker and as such require a more recent version to use plugins as well as the
+Pulumi plugin for Spinnaker, which this guide will show you how to use.
 
-* Spinnaker v1.19.4+
+* Spinnaker v1.19.4+ (version 1.23.x is recommended)
 * Halyard 1.34+
 
-Additionally, your Spinnaker instance should have been deployed using the [distributed installation](https://www.spinnaker.io/setup/install/environment/#distributed-installation) method (i.e. Kubernetes). If you are using any other installation method, although the plugin shown in this document will not work, you may be able to apply similar concepts to your installation.
+**Note**: Refer to the [version compatibility table](https://github.com/pulumi/spinnaker-preconfigured-job-plugin#version-compatibility)
+in the plugin repository.
+
+Additionally, your Spinnaker instance should have been deployed using the [distributed installation](https://www.spinnaker.io/setup/install/environment/#distributed-installation)
+method (i.e. Kubernetes).
 
 ### Pulumi
 
-This guide assumes you are using the default Pulumi Service [managed backend](https://www.pulumi.com/docs/intro/concepts/state/#backends). The Spinnaker plugin referenced in this guide supports specifying alternate backend URLs.
+This guide assumes you are using the default Pulumi Service [managed backend](https://www.pulumi.com/docs/intro/concepts/state/#backends).
 To sign-up for a new Pulumi account, head over to the [Pulumi Console](https://app.pulumi.com/signup).
+The Pulumi plugin for Spinnaker referenced in this guide supports specifying alternate backend URLs.
```

Comment:
My suggestion is to refer to the "Pulumi Plugin for Spinnaker" to help distinguish from generic plugin stuff ... YMMV.  If you agree, the s/r is up to you :)